docs/documentation/stories/rc.0-update.md

```diff
@@ -0,0 +1,451 @@
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
+    ],
+    "outDir": "../dist/out-tsc-e2e",
+    "module": "commonjs",
+    "target": "es6",
+    "types":[
+      "jasmine",
+      "node"
+    ]
+  }
+}
+```
+
+There is an additional root-level `tsconfig.json` that is used for IDE integration.
+```
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "outDir": "./dist/out-tsc",
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
+    ]
+  }
+}
+```
+
+You can delete `e2e/tsconfig.json` and `src/tsconfig.json` after adding these.
+
+Also update `.angular-cli.json` to use them inside `apps[0]`:
+
+```
+"tsconfig": "tsconfig.app.json",
+"testTsconfig": "tsconfig.spec.json",
+```
+
+Then update `protractor.conf.js` to use the e2e config as well:
+```
+beforeLaunch: function() {
+  require('ts-node').register({
+    project: 'e2e'
+  });
+},
+```
+
+## package.json
+
+We've updated a lot of packages over the last months in order to keep projects up to date.
+
+Additions or removals are found in bold below.
+
+Packages in `dependencies`:
+- `@angular/*` packages now have a `^2.4.0` minimum (`^3.4.0` for router)
+- `core-js` remains unchanged at `^2.4.1`
+- `rxjs` to `^5.1.0`
+- `zone.js` to `^0.7.6`
+
+Packages in `dependencies`:
+- `@angular/cli` at `1.0.0-rc.0` replaces `angular-cli`
+- `@angular/compiler-cli` is also at `^2.4.0`
+- `@types/jasmine` remains unchanged and pinned at `2.5.38`
+- `@types/node` was updated to `~6.0.60`
+- `codelyzer` was updated to `~2.0.0`
+- `jasmine-core` was updated to `~2.5.2`
+- `jasmine-spec-reporter` was **removed**
+- `karma` was updated to `~1.4.1`
+- `karma-chrome-launcher` was updated to `~2.0.0`
+- `karma-cli` was updated to `~1.0.1`
+- `karma-jasmine` was updated to `~1.1.0`
+- `karma-jasmine-html-reporter` was **added** at `^0.2.2`
+- `karma-coverage-istanbul-reporter` was **added** at `^0.2.0`, replacing `karma-remap-istanbul`
+- `karma-remap-istanbul` was **removed**
+- `protractor` was updated to `~5.1.0`
+- `ts-node` was updated to `~2.0.0`
+- `tslint` was updated to `~4.4.2`
+- `typescript` was updated to `~2.0.0`
+
+See the [karma](#karma.conf.js) and [protractor](#protractor.conf.js) sections below for more
+information on changed packages.
+
+The [Linting rules](#Linting rules) section contains a list of rules that changed due to updates.
+
+We also updated the scripts section to make it more simple:
+
+```
+"scripts": {
+  "ng": "ng",
+  "start": "ng serve",
+  "test": "ng test",
+  "lint": "ng lint",
+  "e2e": "ng e2e"
+},
+```
+
+## karma.conf.js
+
+Karma configuration suffered some changes to improve the code-coverage functionality,
+use the new `@angular/cli` package, and the new HTML reporter.
```

Comment:
How about this instead, to avoid repeating `use the new` twice:
```
Karma configuration suffered some changes to improve code-coverage functionality, use the new `@angular/cli` package, and report results via HTML reporter.
```